spec/controllers/application_records_controller_spec.rb

```diff
@@ -338,8 +338,8 @@
     context 'record previously saved for later' do
       let!(:record) do
         create :application_record,
-          saved_for_later: true,
-          note_for_later: 'this needs to be here'
+               saved_for_later: true,
```

Comment:
Yes, I just realized there is no comma there. I will make sure to double check the tests.